cPP_DSC.adoc

```diff
@@ -783,137 +791,133 @@ The following table provides the allowed choices for completion of the selection
 
 |KAS2
 |RSA
-|{empty}[*selection*: [.underline]#2048, 3072, 4096, 6144, 8192]# bits
+|[selection: 2048, 3072, 4096, 6144, 8192] bits
 |NIST SP 800-56B Rev 2 (Section 8.3)
 
 |DH
 |Diffie-Hellman
-|{empty}[*selection*: [.underline]#2048, 3072, 4096, 6144, 8192]# bits
-|NIST SP 800-56A Rev 3, {empty}[*selection*: RFC 3526 (Section [.underline]#[*selection*: 3, 4, 5, 6, 7]), RFC 7919 (Appendixes [*selection*: A.1, A.2, A.3, A.4, A.5]#)]
+|[selection: 2048, 3072, 4096, 6144, 8192] bits
+|NIST SP 800-56A Rev 3, [selection: RFC 3526 (Section [selection: 3, 4, 5, 6, 7]), RFC 7919 (Appendixes [selection: A.1, A.2, A.3, A.4, A.5])]
 
 |ECDH-NIST
 |ECDH with NIST curves
-|{empty}[*selection*: [.underline]#256 (P-256), 384 (P-384), 512 (P-521)]#
-|NIST SP 800-56A
+|[selection: 256 (P-256), 384 (P-384), 512 (P-521)] bits
+|NIST SP 800-56Ar3, NIST SP 800-186 (Appendix G.1)
 
 |ECDH-BPC
 |ECDH with Brainpool curves
-|{empty}[*selection*: [.underline]#256 (brainpoolP256r1), 384 (brainpoolP384r1), 512 (brainpoolP512r1)]#
-|RFC 5639 (Section 3)
+|[selection: 256 (brainpoolP256r1), 384 (brainpoolP384r1), 512 (brainpoolP512r1)] bits
+|NIST SP 800-56Ar3, NIST SP 800-186 (Appendix H.1)
 
 |ECDH-Ed
 |ECDH with Edwards Curves
-|{empty}[*selection*: [.underline]#256, 448]# bits
+|[selection: 255, 448] bits
 |RFC 7748
 
 |ECIES
 |ECIES
-|{empty}[*selection*: [.underline]#256, 384, 512]# bits
-|{empty}[*selection*: [.underline]#ANSI X9.63, IEEE 1363a, ISO/IEC 18033-2 Part 2, SECG SEC1 sec 5.1]#
+|[selection: 256, 384, 512] bits
+|[selection: ANSI X9.63, IEEE 1363a, ISO/IEC 18033-2 Part 2, SECG SEC1 sec 5.1]
 
-|KDF
-|{empty}[*selection*: [.underline]#KDF-CTR, KDF-FB, KDF-DPI]# with concatenated keys as input using {empty}[*selection*: [.underline]#AES-128-CMAC; AES-192-CMAC; AES-256-CMAC, HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as the PRF.
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits
-|NIST SP 800-108 (Section 5) [KDF]
+|KAS-KDF-X
```

Comment:
No idea, copied from the catalog. @slpotte

Reply:
This signifies there there are three choices for X, namely -CTR, -FB, and -DPI.  I was worried this might cause confusion.  Would it be better just to leave it as "KAS-KDF" with no suffix?

Reply:
I don't know, could we just list the options in one? I don't know if it makes much difference as long as it is clear (so if normally everyone just talks about KAS-KDF and the details come out in the parameters, I think that is fine)

Reply:
In FIPS, we normally call these KDFs Key-based KDF (KBKDF), because they operate on an input key-derivation-key. Maybe "KAS-KDF-KB" could be a suggestion. Or alternatively, list each of them as a separate row, similar to FCS_CKM.5.1... In the end, I don't really care too much about the -X, I was just curious about its meaning.

Reply:
I'm removing the "-X"

Reply:
I removed the "-X" as well from the catalog.